x-pack/plugins/apm/public/components/app/service_node_metrics/index.tsx

```diff
@@ -42,6 +43,13 @@ const Truncate = styled.span`
   ${truncate(px(unit * 12))}
 `;
 
+const MetadataFlexGroup = styled(EuiFlexGroup)`
+  border-bottom: ${({ theme }) => theme.eui.euiBorderThin};
+  margin-bottom: ${({ theme }) => theme.eui.paddingSizes.m};
+  padding: ${({ theme }) =>
+    `${theme.eui.paddingSizes.m} 0 0 ${theme.eui.paddingSizes.m}`};
+`;
```

Comment:
Would be great if we can use EUI components for this, eg. [horizontal rule](https://elastic.github.io/eui/#/layout/horizontal-rule) and/or [panel](https://elastic.github.io/eui/#/layout/panel).
Not sure if that's a possible alternative though. If not, it's fine as-is.